.env

```diff
@@ -0,0 +1,7 @@
+# Okta variables
```

Comment:
I'm not 100% about this, but on past projects I've been on the `.env` file has been added to `.gitignore` so the secrets aren't pushed up. I think we can create a `.env.sample` (or something along those lines) if we want a template for the environment variables?